templates/adherent/profile.html.twig

```diff
@@ -21,6 +21,7 @@
                     return "{{ 'adherent.form.mandate.no_result'|trans }}";
                 },
             });
+            $('.multi_mandate').prop('required',false);
```

Comment:
I think we need to fix this script instead:
https://github.com/EnMarche/en-marche.fr/blob/master/templates/membership/_mandates_script.html.twig#L11